packages/protocol/script/layer1/DeployProtocolOnL1.s.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.24;
 
 import "@openzeppelin/contracts/utils/Strings.sol";
 import "@risc0/contracts/groth16/RiscZeroGroth16Verifier.sol";
-import { SP1Verifier as SP1Verifier200rc } from "@sp1-contracts/src/v2.0.0/SP1VerifierPlonk.sol";
+import { SP1Verifier as SP1Verifier300rc } from "@sp1-contracts/src/v3.0.0-rc3/SP1VerifierPlonk.sol";
```

Comment:
lets change `SP1Verifier300rc` to `SuccinctVerifeir` so we will not change the name again for newer versions.

Reply:
sure, fix soon.